src/winetricks

```diff
@@ -7,7 +8,20 @@
 # Name of this version of winetricks (YYYYMMDD)
 # (This doesn't change often, use the sha256sum of the file when reporting problems)
 WINETRICKS_VERSION=20180815-next
-
+# added correct install message on any other systems from Ubuntu
+if [ -f /etc/blackPanther-release ];then
+    INSTALLCMD="installing"
+    GUIINSTALLCMD="app-helper"
+elif [ -f /etc/fedora-release -o  -f /etc/redhat-release ];then
+    INSTALLCMD="dnf install"
+    GUIINSTALLCMD="pkcon install"
+elif [ -f /etc/mandrake-release -o -f /etc/mandriva-release -o -f /etc/mangeia-release -o -f /etc/rosa-release ];then
+    INSTALLCMD="urpmi"
+    GUIINSTALLCMD="gurpmi2"
+else 
+    INSTALLCMD="sudo apt-get install"
```

Comment:
Why sudo here and not elsewhere?

Reply:
> `Why sudo here and not elsewhere?

This line just copied from your source. We don't use sudo